irods_testing_environment/ssl_setup.py

```diff
@@ -84,11 +85,72 @@ def generate_ssl_dh_params(generator=2, key_size=1024, directory=None):
     return dhfile
 
 
-def configure_ssl_on_server(container,
-                            path_to_key_file_on_host,
-                            path_to_cert_file_on_host,
-                            path_to_dhparams_file_on_host):
-    """Copy SSL files to the container and configure SSL on the iRODS server.
+def configure_tls_for_service_account(container, cert_file):
+    """Configure TLS for the iRODS service account client environment.
+
+    Arguments:
+    container -- the docker.Container on which SSL will be configured
```

Comment:
I feel that `docker.Container` is intentional. Is that correct?

Reply:
Yes, this is referring to an object of type `Container` from the `docker` module.